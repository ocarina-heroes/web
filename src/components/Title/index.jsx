import styled from 'styled-components'

const H1 = styled.h1`
  margin: 0;
  line-height: 1.15;
  font-size: 4rem;
  text-align: center;
`

export default () => <H1>Welcome to Ocarina Heroes!</H1>
