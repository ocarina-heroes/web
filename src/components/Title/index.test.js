import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Title from '.'

test('it matches snapshot', () => {
  const tree = renderer.create(<Title />).toJSON()
  expect(tree).toMatchSnapshot()
})
