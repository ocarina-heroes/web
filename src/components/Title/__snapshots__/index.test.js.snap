// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
.c0 {
  margin: 0;
  line-height: 1.15;
  font-size: 4rem;
  text-align: center;
}

<h1
  className="c0"
>
  Welcome to Ocarina Heroes!
</h1>
`;
